src/explorer/ConnectionAccountDatabaseTreeItem.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SiteClient } from 'vscode-azureappservice';
+import { IAzureTreeItem } from 'vscode-azureextensionui';
+
+export class ConnectionAccountDatabaseTreeItem implements IAzureTreeItem {
+    public static contextValue: string = 'connectionAccountDatabase';
+    public readonly contextValue: string = ConnectionAccountDatabaseTreeItem.contextValue;
+    public readonly label: string;
+
+    constructor(readonly client: SiteClient, readonly connectionId: string) {
+        this.label = this.getLabel(connectionId);
+    }
+
+    private getLabel(id: string): string {
+        // tslint:disable-next-line:strict-boolean-expressions
+        const items = this.parseCosmos(id) || this.parseAttached(id) || undefined;
```

Comment:
Change the tslint rule here to have 'allow-null-union' and you shouldn't need `|| undefined`

https://github.com/Microsoft/vscode-azureappservice/blob/master/tslint.json#L93

I've been meaning to change that rule anyways.